ci/deploy_app.yaml

```diff
@@ -2,63 +2,16 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: gcr.io/google.com/cloudsdktool/cloud-sdk
-    tag: slim
+    repository: ((image_registry))/eq-infrastructure-deploy-image
+    tag: ((deploy_image_version))
 inputs:
   - name: eq-questionnaire-runner
   - name: image-tag
     optional: true
 params:
   SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  TERRAFORM_STATE_BUCKET:
   PROJECT_ID:
-  DOCKER_REGISTRY:
-  IMAGE_TAG:
-  REGION:
-
-  CONCURRENCY:
-  MIN_INSTANCES:
-  MAX_INSTANCES:
-  CPU:
-  MEMORY:
-
-  WEB_SERVER_TYPE:
-  WEB_SERVER_WORKERS:
-  WEB_SERVER_THREADS:
-  WEB_SERVER_UWSGI_ASYNC_CORES:
-  HTTP_KEEP_ALIVE:
-
-  DATASTORE_USE_GRPC:
-  EQ_STORAGE_BACKEND:
-  EQ_ENABLE_SECURE_SESSION_COOKIE:
-  EQ_RABBITMQ_ENABLED:
-  EQ_ENABLE_HTML_MINIFY:
-  EQ_RABBITMQ_HOST:
-  EQ_RABBITMQ_HOST_SECONDARY:
-  EQ_QUESTIONNAIRE_STATE_TABLE_NAME:
-  EQ_SESSION_TABLE_NAME:
-  EQ_USED_JTI_CLAIM_TABLE_NAME:
-  EQ_SUBMISSION_BACKEND:
-  EQ_FEEDBACK_BACKEND:
-  EQ_PUBLISHER_BACKEND:
-  EQ_SUBMISSION_CONFIRMATION_BACKEND:
-  EQ_FULFILMENT_TOPIC_ID:
-  EQ_INDIVIDUAL_RESPONSE_LIMIT:
-  EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE:
-  EQ_FEEDBACK_LIMIT:
-  COOKIE_SETTINGS_URL:
-  CDN_URL:
-  CDN_ASSETS_PATH:
-  ADDRESS_LOOKUP_API_URL:
-  ADDRESS_LOOKUP_API_AUTH_ENABLED:
-  ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS:
-  NEW_RELIC_ENABLED:
-  CONFIRMATION_EMAIL_LIMIT:
-  GOOGLE_TAG_MANAGER_ID:
-  GOOGLE_TAG_MANAGER_AUTH:
-  NEW_RELIC_LICENSE_KEY:
-  NEW_RELIC_APP_NAME:
-  CLOUD_ARMOR_POLICY_NAME:
-  VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS:
```

Comment:
These had to be dropped from here as it will pass an empty string to TF which mean tf won't use the default since variable is set. If we want to fly execute, we can either add these back and add a default here as well or include the `terrform.tfvars` files when fly executing by including ignored files.